### PR TITLE
uncomment actual enrollment of users and add --do option to keep things backward compatible

### PR DESCRIPTION
**Pull request by Andrey9kin**

### CODE REVIEW

**Comment by Andrey9kin** [comment]
@aren one more, please.

I'm also thinking it would be great to pull data about the user from FoxPass (name, username, etc) and call add_user first. After that enroll the user if a flag specified. Would it be Ok to do that?

File: sync.py [hunk 1]
```diff
@@ -39,7 +39,8 @@
 logger = logging.getLogger(__name__)
 
 parser = argparse.ArgumentParser(description='Sync between Foxpass and Duo.')
-parser.add_argument('--once', action='store_true', help='Run once and exit')
+parser.add_argument('--once', action='store_true', help='Run once and exit (or non-empty FOXPASS_DUO_SYNC_ONCE env. var.)')
+parser.add_argument('--do', action='store_true', help='Run sync otherwise will only print what it would do without actually doing it (or non-empty FOXPASS_DUO_DO_SYNC env. var.)')
```
Comment (aren) [inline sync.py:43]: Normally I see this reversed -- there is a `--dry-run` argument, and the default is to `do`

File: sync.py [hunk 2]
```diff
@@ -98,10 +102,10 @@ def get_all_foxpass_users():
 
 def sync():
     duo_users = admin_api.get_users()
-    duo_email_set = set()
+    duo_email_set = dict()
```
Comment (aren) [inline sync.py:105]: It's not a set anymore. How about `duo_users_by_email = dict()`?

File: sync.py [hunk 3]
```diff
@@ -111,26 +115,54 @@ def sync():
 
     # make a set of foxpass users that should be in duo. they must be active and if a group is
     # specified, they must be in that group
-    foxpass_email_set = set()
+    foxpass_sync_set = list()
```
Comment (aren) [inline sync.py:118]: why switch this from a set to a list? seems to be doing the same thing, but a set is more efficient.

**Comment by aren** [comment]
Much of this is addressed in https://github.com/foxpass/foxpass-duo/pull/4, closing.

**Pull request closed without merging**